README.md

```diff
@@ -97,27 +97,43 @@ The options that we support for calculation are as follows:
 To run the globally installed CLI (see above), use the global `fqm-execution command`
 
 ```bash
-Usage: fqm-execution [options]
+Usage: fqm-execution [command] [options]
+
+Commands:
+ detailed (default)
+ raw
+ reports
+ gaps
+ dataRequirements
+ queryInfo
```

Comment:
Do we want more description here? Previous entry for output types did not elaborate. Do these output types speak for themselves?